src/tasks/package.js

```diff
@@ -74,6 +84,23 @@ module.exports = (config) => {
         'ci:lint': 'npm run lint && npm run security',
         'ci:test': 'npm run test -- --runInBand',
         'ci:coverage': 'npm run test:coverage -- --runInBand',
+        'webpack-defaults': 'webpack-defaults',
+      },
+      dependencies: existingPackage.dependencies || {},
+      devDependencies: existingPackage.devDependencies || {},
+      engines: {
```

Comment:
I would personally move `pkg.engines` above `pkg.main`, but I get the point to place it near the `peerDependencies`, also not important

````
name: ''.
version: '',
author: '',
description: '',
license: '',
engines: '',
main: '',
files: '',
dependencies: '',
devDependencies: '',
peerDependencies: '',
scripts: '',
homepage: '',
repository: '', 
bugs: '',
// ...third-party pre-commit, lint-staged etc
```